### PR TITLE
Link snowflake account

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Updating Snowflake Native App with instructions for linking accounts

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud-integrations/about-snowflake-native-app.md [hunk 1]
```diff
@@ -34,7 +34,8 @@ The following diagram provides an illustration of the architecture:
 
 
 ## Access
-You can log in to the dbt Snowflake Native App using your regular Snowflake login authentication method. During this [Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud), you do not need dbt Cloud credentials (a dbt Cloud seat) to access the application but this is subject to change. 
+
+Log in to the dbt Snowflake Native App using your regular Snowflake login authentication method. The Snowflake user must have a corresponding dbt Cloud user. Previously, this wasn't a requirement during the feature [Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud). If your account is already configured, you will be prompted to [link credentials](#link-credentials) the next time you access dbt Cloud from the Snowflake app, and if you don't have dbt Cloud credentials, the integration will no longer work for your user.
```
Comment (amychen1776) [inline website/docs/docs/cloud-integrations/about-snowflake-native-app.md]: >If your account is already configured, you will be prompted to [link credentials](#link-credentials) the next time you access dbt Cloud from the Snowflake app, and if you don't have dbt Cloud credentials, the integration will no longer work for your user.

This sentence is a bit confusing to me. Do you mean if you have already configured your native app to dbt cloud, you can just link your user once and we will store it so you don't have to do it again? Or that if it's not configured or you don't have an dbt cloud seat - you can't access it?

Comment (matthewshaver) [reply website/docs/docs/cloud-integrations/about-snowflake-native-app.md]: Yeah, that's a rough sentence. Let me clear that up

File: website/docs/docs/cloud-integrations/about-snowflake-native-app.md [hunk 2]
```diff
@@ -34,7 +34,12 @@ The following diagram provides an illustration of the architecture:
 
 
 ## Access
-You can log in to the dbt Snowflake Native App using your regular Snowflake login authentication method. During this [Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud), you do not need dbt Cloud credentials (a dbt Cloud seat) to access the application but this is subject to change. 
+
+Log in to the dbt Snowflake Native App using your regular Snowflake login authentication method. The Snowflake user must have a corresponding dbt Cloud user with a _[developer license](/docs/cloud/manage-access/seats-and-users)_. Previously, this wasn't a requirement during the feature [Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud). 
+
+If your Snowflake Native App is already configured, you will be prompted to [link credentials](#link-credentials) the next time you access dbt Cloud from the app. This is a one-time process. If you don't have a dbt Cloud account associated with a developer license, you will be denied access to the dbt Cloud environment and will need an admin to create the account.
```
Comment (amychen1776) [inline website/docs/docs/cloud-integrations/about-snowflake-native-app.md]:
```suggestion
If your Snowflake Native App is already configured, you will be prompted to [link credentials](#link-credentials) the next time you access dbt Cloud from the app. This is a one-time process. If you don't have a dbt Cloud account associated with a developer license, you will be denied access to the dbt Cloud environment and will need an admin to assist.
```

File: website/docs/docs/cloud-integrations/about-snowflake-native-app.md [hunk 3]
```diff
@@ -47,4 +52,18 @@ If you're interested, please [contact us](matilto:sales_snowflake_marketplace@db
...
-- The Native app does not support dbt Cloud accounts with [IP Restrictions](/docs/cloud/secure/ip-restrictions] enabled. 
+- The Native app does not support dbt Cloud accounts with [IP Restrictions](/docs/cloud/secure/ip-restrictions) enabled. 
+
+## Link credentials
+
+During the early stages of the Snowflake Native App preview, users were only required to exist in the Snowflake platform and could access dbt Cloud via the app without having a corresponding user. This is no longer the case, and every Snowflake user must also have dbt Cloud account access with a [developer license](/docs/cloud/manage-access/seats-and-users). 
+
+For existing accounts with the Snowflake Native App configured, users will be prompted to authenticate with dbt Cloud the next time they log in. This is a one-time process if they have a user in dbt Cloud. If they don’t have a dbt Cloud user, they will be denied access, and an admin will need to [create one](/docs/cloud/manage-access/invite-users). 
+
+1. When you attempt to access the dbt Cloud platform from the Snowflake Native App, you will be prompted to link your accounts.
```
Comment (amychen1776) [inline website/docs/docs/cloud-integrations/about-snowflake-native-app.md]:
```suggestion
1. When you attempt to access the dbt Cloud platform from the Snowflake Native App, you will be prompted to link your account.
```

**Review by amychen1776 — APPROVED** [review]
just a few comments but outside of that, ready to ship

**Pull request merged**